cnm/network/network.go

```diff
@@ -173,7 +174,7 @@ func (plugin *netPlugin) createNetwork(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	err = plugin.nm.CreateNetwork(&nwInfo)
+	err = plugin.nm.CreateNetwork(&nwInfo, &cni.NetworkConfig{})
```

Comment:
with platformInit change, i dont think its required to pass networkconfig...is that right? if so, can you please remove passing cni config from all places